doc/source/user_guide/categorical.rst

```diff
@@ -625,8 +625,8 @@ even if some categories are not present in the data:
     df = pd.DataFrame(
         data=[[1, 2, 3], [4, 5, 6]],
         columns=pd.MultiIndex.from_arrays([["A", "B", "B"], columns]),
-    )
-    df.groupby(axis=1, level=1).sum()
+    ).T
+    df.groupby(level=1).sum()
```

Comment:
I think this whole example can also be removed; it started as `DataFrame.sum` with `level=1` passed which used groupby under the hood, making the unused categories show up (observed=False is the default). Now `level` is not an argument do `DataFrame.sum`, and so the docs here are not correct - unused categories don't show up in `DataFrame.sum`.

Groupby is shown in the next example, which I think is sufficient.

But potentially removing this example would also be okay for a different PR.

Reply:
> But potentially removing this example would also be okay for a different PR.

I'd like to punt on this for now, yah